tests/frontend-storage/cypress/integration/FrontendStorage.test.ts

```diff
@@ -93,3 +95,69 @@ describe(`${FrontendStorage.name}: ${frontendStorage.constructor.name}`, () => {
     });
   });
 });
+
+/**
+Common frontend UNIT tests.
+Should be moved to a separate frontend-storage-unit package in the future,
+but it isn't optimal to do it at the moment.
+*/
+describe(`${FrontendStorage.name}: ${frontendStorage.constructor.name} (Unit)`, () => {
```

Comment:
I'd suggest to not postfix it with `(Unit)` as the tests are not unit, the code flow just does not hit the statements that make http calls because of errors thrown in validation but a test does not become a unit test because of that. Maybe a better identification would be to add postfix `(Input validation)`?